app/components/certificate/description.tsx

```diff
@@ -0,0 +1,36 @@
+import { Flex, Heading, Text, HStack, VStack } from '@chakra-ui/react';
+import type { Certificate } from '@prisma/client';
+
+interface Props {
+  request: Boolean;
+  cert: Certificate;
+}
+
+export default function Description(props: Props) {
+  return (
+    <Flex flexDirection="column" gap="3" marginTop="14">
+      <Heading as="h1" size="xl">
+        Certificate
+      </Heading>
+      <Text>
+        {props.request
+          ? "Certificate: Lorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum has been the industry's standard dummy text ever since the 1500s"
+          : "Initial: Lorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum has been the industry's standard dummy text ever since the 1500s"}
+      </Text>
+      {props.request ? (
+        <HStack gap="6" marginTop="2">
+          <VStack>
+            <Text fontWeight="bold">Created On</Text>
+            <Text>{props.cert.validFrom.toDateString()}</Text>
+          </VStack>
+          <VStack>
+            <Text fontWeight="bold">Expires On</Text>
+            <Text>{props.cert.validTo.toDateString()}</Text>
+          </VStack>
+        </HStack>
+      ) : (
+        ''
```

Comment:
Since you have nothing to return in the `else` case I think it is better to use `and` operator instead of the ternary one. So it would be just 
```typescript
{props.request && <SomethingYouWant />
```

Reply:
That's true